packages/cursorless-vscode-e2e/src/suite/fixtures/recorded/actions/snippets/customWrapLine2.yml

```diff
@@ -0,0 +1,32 @@
+languageId: plaintext
+command:
+  version: 5
+  spokenForm: custom wrap line
+  action:
+    name: wrapWithSnippet
+    args:
+      - {type: custom, body: 'dummy snippet hole1: ($TM_SELECTED_TEXT), hole2: ($hole2)'}
```

Comment:
Do we want to use `TM_SELECTED_TEXT`, or should we make up our own variable name?

Reply:
I think it's fine.

Reply:
It's just a bit confusing because it's not actually selected text that we're targeting; it's the target.  I'd be tempted to do `$CURSORLESS_TARGET`.  But I guess using `$TM_SELECTED_TEXT` means that we'll work out of the box for any wrapper-style snippets people already have

Reply:
let's leave it for now